<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>451</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>20</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_userinfo">
     <attribute name="title">
      <string>用户</string>
     </attribute>
     <widget class="QScrollBar" name="horizontalScrollBar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>370</y>
        <width>661</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBar">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>0</y>
        <width>16</width>
        <height>371</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>280</y>
        <width>531</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>启动时间---------------------------------------------------------------------------------------------------------</string>
      </property>
     </widget>
     <widget class="QLabel" name="host_Label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>240</y>
        <width>341</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>531</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>主机名-----------------------------------------------------------------------------------------------------------</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>版本--------------------------------------------------------------</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>531</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>系统---------------------------------------------------------------------------------------------------------------</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_version">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_cpu">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>351</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_memory">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_runtime">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_graph">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>191</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_function">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>性能</string>
     </attribute>
     <widget class="QScrollBar" name="horizontalScrollBar_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>370</y>
        <width>661</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBar_2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>0</y>
        <width>16</width>
        <height>371</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarCpu">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarMem">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>65</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cpu Rate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>200</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Memrory Rate</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_pid">
     <attribute name="title">
      <string>进程</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>681</width>
        <height>381</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>132</number>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_action">
     <property name="maximumSize">
      <size>
       <width>16777214</width>
       <height>16777214</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <attribute name="title">
      <string>UserAction</string>
     </attribute>
     <widget class="QScrollBar" name="horizontalScrollBar_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>370</y>
        <width>661</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBar_4">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>0</y>
        <width>16</width>
        <height>371</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>261</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cp">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>70</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_exit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>270</y>
        <width>291</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
